pkg/api/external/kubernetes/customresourcedefinition/customresourcedefinition_base_client_test.go

```diff
@@ -19,7 +19,7 @@ import (
 
 var _ = Describe("DeploymentBaseClient", func() {
 	if os.Getenv("RUN_KUBE_TESTS") != "1" {
-		log.Printf("This test creates kubernetes resources and is disabled by default. To enable, set RUN_KUBE_TESTS=1 in your env.")
+		fmt.Print("This test creates kubernetes resources and is disabled by default. To enable, set RUN_KUBE_TESTS=1 in your env.")
```

Comment:
I originally did this to avoid referencing k8s-utils in our tests. This is because the k8s library still was using ginkgov1 wand that causes tests to panic (called out in last section of PR). I can actually revert this, as I ended up updating k8s-utils as part of this work